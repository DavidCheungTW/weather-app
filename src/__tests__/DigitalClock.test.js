import React from "react";
import DigitalClock from "../components/DigitalClock";
import renderer from "react-test-renderer";

describe("DigitalClock", () => {
  test("Renders as expected", () => {
    Date.now = jest.fn(() => 1680964077162);

    const rendered = renderer.create(<DigitalClock />);

    expect(rendered).toMatchSnapshot();
  });
});
