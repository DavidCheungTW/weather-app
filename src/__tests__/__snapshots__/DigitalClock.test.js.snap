// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DigitalClock Renders as expected 1`] = `
<div
  className="digital-clock"
>
   
  3:27:57 PM
   
</div>
`;
